CMakeLists.txt

```diff
@@ -470,7 +470,7 @@ if(BUILD_ONNX_PYTHON)
     set_target_properties(onnx_cpp2py_export
                           PROPERTIES LINK_FLAGS "-undefined dynamic_lookup")
     target_link_libraries(onnx_cpp2py_export
-                          PRIVATE -Wl,-force_load,$<TARGET_FILE:onnx>)
+                          PRIVATE -Wl,-force_load,"$<TARGET_FILE:onnx>")
```

Comment:
Note: I only put double quotes around $<TARGET_FILE:onnx> for Mac. Other platforms like Windows and Ubuntu originally work fine from my experiments and further put double quotes will bring other issues. Therefore I didn't put double quotes for other platforms.

Reply:
LGTM. Could you please also write the note above this line in CMakeLists.txt so that future readers can know the context?

Reply:
Good idea. Just added. Thanks!